pkg/syncer/shared/namespace.go

```diff
@@ -105,3 +106,7 @@ func PhysicalClusterNamespaceName(l NamespaceLocator) (string, error) {
 	// keep the namespaces short enough.
 	return fmt.Sprintf("kcp-%s", base36hash[:12]), nil
 }
+
+func IsNamespace(gvr schema.GroupVersionResource) bool {
+	return gvr.Resource == "namespaces" && gvr.Group == "" && gvr.Version == "v1"
+}
```

Comment:
what's the value of this function? just compare with `corev1.Resource("namespaces")`.

Reply:
Fixed.